google_auth_oauthlib/tool/__main__.py

```diff
@@ -0,0 +1,108 @@
+# Copyright (C) 2017 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Command-line tool for obtaining authorization and credentials from a user.
+
+This tool uses the OAuth 2.0 Authorization Code grant as described in
+`section 1.3.1 of RFC 6749`__ and implemeted by
+:class:`google_auth_oauthlib.flow.Flow`.
+
+This tool is intended to bootstrap development in situation when the
+application cannot easily run the 3LO OAuth2 flow, i.e: in an embedded
```

Comment:
super nit: 3LO is never defined in the rfc, so please let's stick to terminology used in the rfc. Maybe:

> This tool is intended for assist developers in obtaining credentials for testing applications where it may not be possible or easy to run a complete OAuth 2.0 authorization flow, especially in the case of code samples or embedded devices without input / display capabilities.